Resources/views/Calendar/view.html.twig

```diff
@@ -119,7 +119,15 @@
                             <hr/>
                         </div>
                         {% endif %}
-                        <div><label class="bold">{{'global.stop_point_code'|trans({}, 'messages')}}</label>: {{ calendar.schedules.stop_point.codes|externalCode }}</div>
+                        <div>
+                            <dl>
+                                <dt>{{'global.stop_point_code'|trans({}, 'messages')}}</dt>
+                                {% for code in calendar.schedules.stop_point.codes %}
+                                {% set value = code.type == 'external_code' ? code.value|slice(3) : code.value %}
```

Comment:
Ok but it's good to be clear with it.
In case, if we need the real codes (with 3 digits) of StopPoint entity, if you change the real value returned by navitia in the setter of codes, it will be forget next time you call the getter, and the data will be false. You see ?